### PR TITLE
format code

### PR DESCRIPTION
**Pull request by schjan**
Format code according to [giantswarm/fmt](https://github.com/giantswarm/fmt) and better handle some former unhandled errors.

_Part of giantswarm/giantswarm#6145_

### CODE REVIEW

File: config/config_test.go [hunk 1]
```diff
@@ -43,15 +43,15 @@ func tempConfig(fs afero.Fs, configYAML string) (string, error) {
 	return dir, nil
 }
 
-// Test_Initialize_Empty tests the case where a config file and its directory
+// TestInitializeEmpty tests the case where a config file and its directory
```
Comment (xh3b4sd) [inline config/config_test.go]: I think this is not better. `fmt` also defines it with the underscores.

File: oidc/pkce.go [hunk 2]
```diff
@@ -160,16 +160,21 @@ func getToken(code, codeVerifier string) (pkceResponse PKCEResponse, err error)
...
 	if err != nil {
 		pkceResponse.Error = "unknown error"
 		pkceResponse.ErrorDescription = "Got an unparseable error from Auth0. Possibly the Auth0 service is down. Try again later."
 		return pkceResponse, microerror.Maskf(authorizationError, pkceResponse.Error)
 	}
 
-	json.Unmarshal(body, &pkceResponse)
+	err = json.Unmarshal(body, &pkceResponse)
+	if err != nil {
+		pkceResponse.Error = "unknown error"
```
Comment (MarcelMue) [inline oidc/pkce.go]: These error functions / descriptors look weird - not sure if it's something to worry about here.

File: oidc/jwt.go [hunk 3]
```diff
@@ -43,9 +46,13 @@ func ParseIDToken(tokenString string) (token *IDToken, err error) {
 		return result, nil
 	})
 	if err != nil {
-		// handle some validation errors specifically
+		// handle some validation errors specifically.
 		valErr, valErrOK := err.(*jwt.ValidationError)
 		if valErrOK && valErr.Errors == jwt.ValidationErrorIssuedAt {
+			if t == nil {
```
Comment (MarcelMue) [inline oidc/jwt.go]: Can `t` actually be nil from the call above?

File: oidc/pkce.go (hunk 2 above)
Comment (schjan) [reply oidc/pkce.go]: changing that `PKCEResponse` struct would also end in changes in `gsctl`. That should definitely be done in a separate PR.

**Pull request merged**